src/index.ts

```diff
@@ -1,49 +1,66 @@
-#!/usr/bin/env node --es-module-specifier-resolution=node
+#!/usr/bin/env node 
 
 import chalk from 'chalk'
-import figlet from 'figlet'
 import { Command } from 'commander'
 
 import { loadProcesses } from './lib/process/index.js'
 
+const { log } = console
+const version = '1.4.0'
```

Comment:
Can't you get version directly from `package.json` ?

Reply:
I can not due to configuration, getting some asset error and suggestion is to `import { version } from package.json as asset` or something. Researched already, something to do with module resolution and our config. When I setup a new project I was able, to be fair it might be easier to start a new one then investigate this one. If you have any ideas I would be happy to try.

Reply:
FYI:

Reply:
this can be done and I'm looking for where I've done it before

Reply:
ok not typescript but the correct javascript import is like https://github.com/digicatapult/dscp-identity-service/blob/e89c181c9aec16dce45e191c74dca799d32c62b8/app/version.js#L1

You then need the right parser options in for eslint https://github.com/digicatapult/dscp-identity-service/blob/e89c181c9aec16dce45e191c74dca799d32c62b8/.eslintrc#L9

Not quite the same but this is doable

Reply:
For .js yes, but for ts there is more, to be fair I would not mind to start a new, but just worries about the review time. Would you like me to investigate this further?

I rather leave this for Friday or some other day, but happy to dive in and get to the bottom. It was working fine when I setup a new project, so have a feeling that it's either a couple of mins or a couple of hours if not more. And might require more changes due to directory configuration.